src/control-plane/control-plane-api.ts

```diff
@@ -26,7 +27,7 @@ export class ControlPlaneAPI extends Construct {
   public readonly tenantUpdateServiceTarget: targets.ApiGateway;
   constructor(scope: Construct, id: string, props: ControlPlaneAPIProps) {
     super(scope, id);
-
+    addTemplateTag(this, 'ControlPlaneAPI');
```

Comment:
The ControlPlaneAPI currently is tightly coupled with the ControlPlane such that you can't use one without the other, right? If that's the case, do we need this?

Reply:
Removed